### DISCUSSION TITLE
Can I make two settings on one controller and switch between them with one click? (for pc-engine)

### DISCUSSION THREAD

**Discussion by szuping** [darthcloud/BlueRetro#839] [General]
![M30-2](https://github.com/darthcloud/BlueRetro/assets/150599431/d8707e67-da60-4731-811f-b2f13f787dcc)
Also, I found that when I use "MultiTap Mode" on PC-Engine, it will not work with the hotkeys of Turbo EveryDrive Pro, and all hotkeys of TEP will become invalid. Is this normal?

**Comment by darthcloud** [comment]
No you can't, BlueRetro only got a single config.

So both mode will be affected if you customize it.

**Reply by szuping** [reply]
Yes, switching the first setting (2button) will indeed affect the second setting (6button). I thought it was because I was not familiar with the setting. Thank you for your reply.

**Comment by darthcloud** [comment]
Btw the Everdrive Pro hotkeys do not even work when using a wired multitap and regular controller. Nothin I can do on my side. It's something Krikzz need to fix.

When BlueRetro is in regular mode it works.

**Reply by szuping** [reply]
I understand, I will report it to Krikzz, thank you very much.
